portfolio/src/main/webapp/script.js

```diff
@@ -15,6 +15,7 @@
 /**
  * Adds a random greeting to the page.
  */
+/** 
 function addRandomGreeting() {
```

Comment:
Same comment as earlier: It's helpful to delete commented code rather than leave it in.

Reply:
Edited.